src/resources/Enums.ts

```diff
@@ -653,6 +653,31 @@ export enum ResultRankingPredicateKind {
     qplPredicate = 'qplPredicate',
 }
 
+export enum PredicateKind {
```

Comment:
If we want to add this one, we need to add a a deprecated for `ResultRankingPredicateKind` because right now we have a duplicate enum

Reply:
fair enough thanks !!